Index: .dockerignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.dockerignore b/.dockerignore
new file mode 100644
--- /dev/null	(date 1778331445641)
+++ b/.dockerignore	(date 1778331445641)
@@ -0,0 +1,25 @@
+**/.dockerignore
+**/.env
+**/.git
+**/.gitignore
+**/.project
+**/.settings
+**/.toolstarget
+**/.vs
+**/.vscode
+**/.idea
+**/*.*proj.user
+**/*.dbmdl
+**/*.jfm
+**/azds.yaml
+**/bin
+**/charts
+**/docker-compose*
+**/Dockerfile*
+**/node_modules
+**/npm-debug.log
+**/obj
+**/secrets.dev.yaml
+**/values.dev.yaml
+LICENSE
+README.md
\ No newline at end of file
Index: .idea/.idea.Gateway/.idea/awsToolkit.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.idea.Gateway/.idea/awsToolkit.xml b/.idea/.idea.Gateway/.idea/awsToolkit.xml
new file mode 100644
--- /dev/null	(date 1778331504246)
+++ b/.idea/.idea.Gateway/.idea/awsToolkit.xml	(date 1778331504246)
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="toolkitAccountSettings">
+    <option name="activeRegion" value="us-east-1" />
+    <option name="recentlyUsedRegions">
+      <list>
+        <option value="us-east-1" />
+      </list>
+    </option>
+  </component>
+</project>
\ No newline at end of file
Index: DF.Gateway.API/Dockerfile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/DF.Gateway.API/Dockerfile b/DF.Gateway.API/Dockerfile
new file mode 100644
--- /dev/null	(date 1778331445609)
+++ b/DF.Gateway.API/Dockerfile	(date 1778331445609)
@@ -0,0 +1,23 @@
+FROM mcr.microsoft.com/dotnet/aspnet:10.0 AS base
+USER $APP_UID
+WORKDIR /app
+EXPOSE 8080
+EXPOSE 8081
+
+FROM mcr.microsoft.com/dotnet/sdk:10.0 AS build
+ARG BUILD_CONFIGURATION=Release
+WORKDIR /src
+COPY ["DF.Gateway.API/DF.Gateway.API.csproj", "DF.Gateway.API/"]
+RUN dotnet restore "DF.Gateway.API/DF.Gateway.API.csproj"
+COPY . .
+WORKDIR "/src/DF.Gateway.API"
+RUN dotnet build "./DF.Gateway.API.csproj" -c $BUILD_CONFIGURATION -o /app/build
+
+FROM build AS publish
+ARG BUILD_CONFIGURATION=Release
+RUN dotnet publish "./DF.Gateway.API.csproj" -c $BUILD_CONFIGURATION -o /app/publish /p:UseAppHost=false
+
+FROM base AS final
+WORKDIR /app
+COPY --from=publish /app/publish .
+ENTRYPOINT ["dotnet", "DF.Gateway.API.dll"]
Index: .idea/.idea.Gateway/.idea/indexLayout.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.idea.Gateway/.idea/indexLayout.xml b/.idea/.idea.Gateway/.idea/indexLayout.xml
new file mode 100644
--- /dev/null	(date 1778331504232)
+++ b/.idea/.idea.Gateway/.idea/indexLayout.xml	(date 1778331504232)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="UserContentModel">
+    <attachedFolders />
+    <explicitIncludes />
+    <explicitExcludes />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.idea.Gateway/.idea/projectSettingsUpdater.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.idea.Gateway/.idea/projectSettingsUpdater.xml b/.idea/.idea.Gateway/.idea/projectSettingsUpdater.xml
new file mode 100644
--- /dev/null	(date 1778331504238)
+++ b/.idea/.idea.Gateway/.idea/projectSettingsUpdater.xml	(date 1778331504238)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="RiderProjectSettingsUpdater">
+    <option name="singleClickDiffPreview" value="1" />
+    <option name="unhandledExceptionsIgnoreList" value="1" />
+    <option name="vcsConfiguration" value="3" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.idea.Gateway/.idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.idea.Gateway/.idea/vcs.xml b/.idea/.idea.Gateway/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1778331504225)
+++ b/.idea/.idea.Gateway/.idea/vcs.xml	(date 1778331504225)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$/../.." vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
